pipelines/batch/src/main/java/com/google/fhir/analytics/FhirSearchUtil.java

```diff
@@ -156,13 +161,15 @@ public String findSelfUrl(Bundle searchBundle) {
     return selfLink;
   }
 
-  private IQuery<Bundle> makeQueryForResource(String resourceType, int count) {
+  private IQuery<Bundle> makeQueryForResource(String resourceType, int count, int offset) {
     IGenericClient client = openmrsUtil.getSourceClient(true);
     return client
         .search()
         .forResource(resourceType)
         .totalMode(SearchTotalModeEnum.ACCURATE)
         .count(count)
+        .offset(offset)
```

Comment:
The problem is that adding this parameter, breaks OpenMRS's implementation of FHIR. It seems to be a bug on their side but regardless what happens is that once `_offset` parameter is present, it tries to return all resources (i.e, no pagination) and that causes the time-out issue we are seeing in the e2e test.